application.conf

```diff
@@ -43,4 +43,12 @@ openam {
   testUser=""
   testUserPassword=""
   tokenUrl=""
+  deploymentUri = "replace_with_openam_deployment_uri"
+  realm = "/apollo"
+  username = "replace_with_openam_username"
+  password = "replace_with_openam_password"
+  authIndex {
+    type = "module"
+    value = "DataStore"
+  }
```

Comment:
Will create another ticket for this. From my understanding, we do need something like you describe above for logging to work.

A related issue: our unit tests based on the spray-testkit examples only access inner routes directly with a TestActor. We don't have integration tests of `VaultServiceActor.runRoute()`. So often during testing I would NOT see logs I expected since we never actually create the `VaultServiceActor`.